exercises/book-store/description.md

```diff
@@ -1,3 +1,5 @@
+To try and encourage more sales of different books from a popular 5 book series, a bookshop has decided to offer discounts of multiple-book purchases.
+
```

Comment:
It seems these lines are duplicated below, albeit with a slightly different wording.